PluginsAndFeatures/azure-toolkit-for-intellij/azure-sdk-reference-book/src/main/java/com/microsoft/azure/toolkit/intellij/azuresdk/service/AzureSdkLibraryService.java

```diff
@@ -5,98 +5,47 @@
 
 package com.microsoft.azure.toolkit.intellij.azuresdk.service;
 
-import com.fasterxml.jackson.databind.MappingIterator;
+import com.fasterxml.jackson.databind.DeserializationFeature;
 import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.dataformat.csv.CsvMapper;
-import com.fasterxml.jackson.dataformat.csv.CsvSchema;
-import com.microsoft.azure.toolkit.intellij.azuresdk.model.AzureSdkArtifactEntity;
-import com.microsoft.azure.toolkit.intellij.azuresdk.model.AzureSdkFeatureEntity;
-import com.microsoft.azure.toolkit.intellij.azuresdk.model.AzureSdkPackageEntity;
+import com.fasterxml.jackson.databind.ObjectReader;
+import com.fasterxml.jackson.dataformat.yaml.YAMLMapper;
 import com.microsoft.azure.toolkit.intellij.azuresdk.model.AzureSdkServiceEntity;
 import lombok.Getter;
-import org.apache.commons.collections.ListUtils;
-import org.apache.commons.lang3.StringUtils;
-import org.apache.commons.lang3.tuple.Pair;
 
 import java.io.IOException;
 import java.net.URL;
 import java.util.ArrayList;
+import java.util.Collections;
 import java.util.List;
-import java.util.stream.Collectors;
 
 public class AzureSdkLibraryService {
-    private static final ObjectMapper mapper = new CsvMapper();
-    private static final String SDK_METADATA_URL = "https://raw.githubusercontent.com/Azure/azure-sdk/master/_data/releases/latest/java-packages.csv";
-
-    @Getter
-    private List<AzureSdkServiceEntity> serviceEntities = new ArrayList<>();
-    @Getter
-    private List<AzureSdkArtifactEntity> artifactEntities = new ArrayList<>();
+    private static final ObjectMapper mapper = new YAMLMapper().configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
```

Comment:
If service update the schema, shall we skip the unknow properties exception? Or library service will break until we fix the model.

Reply:
just skip.
this config is to skip unused properties.